winbuild/get_pythons.py

```diff
@@ -3,7 +3,7 @@
 from fetch import fetch
 
 if __name__ == "__main__":
-    for version in ["2.7.15", "3.4.4"]:
```

Comment:
Python 3.4.4 is also no longer supported. Is `winbuild/get_pythons` actually used anywhere?

Reply:
It's not used anywhere but is mentioned in and `./winbuild/build.rst` and `winbuild/README.md`. Is it useful for local use?

Reply:
Given that you have to install each downloaded file manually anyway, I doubt it's very useful. And given that nobody has noticed the lack of new versions, it sounds like nobody uses it either.

Looking at its git history, it was added in https://github.com/python-pillow/Pillow/commit/590691d23d96fa15a41d94e4f446afae3de358d2 back in 2014, and was already out of date in 2015 when it was merged in: https://github.com/python-pillow/Pillow/pull/1278#discussion_r32582974

Based on this, I would just remove it completely.

Reply:
Let's do that in a separate PR, it's not strictly related to this one which is already quite big :)

Reply:
I mentioned wanting to re-do the `winbuild` stuff in #4084, perhaps this can be part of that.